shared/templates/template_BASH_audit_rules_dac_modification

```diff
@@ -11,8 +11,15 @@ for ARCH in "${RULE_ARCHS[@]}"
 do
 	PATTERN="-a always,exit -F arch=$ARCH -S %ATTR%.*"
 	GROUP="perm_mod"
+{{% if product != "rhel6" %}}
 	FULL_RULE="-a always,exit -F arch=$ARCH -S %ATTR% -F auid>=1000 -F auid!=4294967295 -F key=perm_mod"
+{{% else %}}
+	FULL_RULE="-a always,exit -F arch=$ARCH -S %ATTR% -F auid>=500 -F auid!=4294967295 -k perm_mod"
+{{% endif %}}
+
+{{% if product != "rhel6" %}}
 	# Perform the remediation for both possible tools: 'auditctl' and 'augenrules'
-	fix_audit_syscall_rule "auditctl" "$PATTERN" "$GROUP" "$ARCH" "$FULL_RULE"
 	fix_audit_syscall_rule "augenrules" "$PATTERN" "$GROUP" "$ARCH" "$FULL_RULE"
```

Comment:
same here, one little part of it is different, we shouldn't repeat the whole line because of it.